packages/rml/rml.1.09.06/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+homepage: "http://reactiveml.org"
+maintainer: "devel@reactiveml.org"
+bug-reports: "bugs@reactiveml.org"
+build: [
+["./configure" "--prefix" "%{prefix}%"]
+[make]
+]
+install: [
+ [make "install"]
+]
+remove: [
+ ["./configure" "--prefix" "%{prefix}%"]
+ [make "uninstall"]
+]
```

Comment:
The `remove` section is unnecessary since opam 2.0, unless you are also removing files added at runtime such as config files. Could you remove this section?